### PR TITLE
Updated Documentation of README.md

### PR DESCRIPTION
**Pull request by Shivam7-1**
**Changes**: Updated Documentation of README.md

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [ ] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [ ] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [ ] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Repository for the PSLab Android App for performing experiments with the [Pocket
 [![Gitter](https://badges.gitter.im/fossasia/pslab.svg)](https://gitter.im/fossasia/pslab?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 [![Twitter Follow](https://img.shields.io/twitter/follow/pslabio.svg?style=social&label=Follow&maxAge=2592000?style=flat-square)](https://twitter.com/pslabio)
 
-This repository holds the Android App for performing experiments with [PSLab](https://pslab.io/). PSLab is a tiny pocket science lab that provides an array of equipment for doing science and engineering experiments. It can function like an oscilloscope, waveform generator, frequency counter, programmable voltage and current source and also as a data logger. Our website is at https://pslab.io
+This repository holds the Android App for performing experiments with [PSLab](https://pslab.io/). PSLab is a tiny pocket science lab that provides an array of equipment for doing science and engineering experiments. It can function like an oscilloscope, waveform generator, frequency counter, gyroscope, compass, programmable voltage and current source and also as a data logger. Our website is at https://pslab.io
```
Comment (CloudyPadmal) [inline README.md:14]: Here it's talking about the PSLab device; not the app.

**Comment by CloudyPadmal** [comment]
@Shivam7-1 please squash the commits into one

**Comment by Shivam7-1** [comment]
How to squash commits in windows

**Comment by CloudyPadmal** [comment]
Instructions are mentioned in the readme file

File: README.md [hunk 2]
```diff
@@ -199,7 +199,7 @@ To use PSLab device with Android, you simply need an OTG cable, an Android Devic
 
 Please help us follow the best practice to make it easy for the reviewer as well as the contributor. We want to focus on the code quality more than on managing pull request ethics.
 
- * Single commit per pull request
+ * Single commit per pull request.
```
Comment (CloudyPadmal) [inline README.md:202]: See, you have made an edit here. But didn't follow it..

**Comment by Shivam7-1** [comment]
I am Unable to squash all commit can you tell how to squash

**Pull request merged**